sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -141,6 +143,38 @@ object DottyPlugin extends AutoPlugin {
   }
 
   override def projectSettings: Seq[Setting[_]] = {
+    // get plugins
```

Comment:
Add comments explaining that this is temporary until we fix sbt, with link to the relevant sbt issue.